crates/store/src/state.rs

```diff
@@ -52,8 +53,12 @@ struct InnerState {
 
 /// The rollup state
 pub struct State {
+    /// The underlying database.
     db: Arc<Db>,
 
+    /// The underlying block store.
+    block_store: Arc<BlockStore>,
```

Comment:
I would change the comment to something like: "The block store which stores full block contents for all blocks."